PIM-8542: Counter on categories is not being updated when deleting a product on the grid (#10382)
akeneo/pim-community-dev: CHANGELOG-3.1.md

# 3.1.x

 ## Bug fixes

>  - PIM-8542: Counter on categories is not being updated when deleting a product on the grid

Reviewer: ```suggestion
 - PIM-8542: Fix counter on categories when deleting a product on the grid
```
